scripts/test/run-tests.sh

```diff
@@ -64,16 +64,16 @@ else
                      "release-1" \
                      "${UBUNTU_IMAGE_URL}" \
                      "${UBUNTU_IMAGE}" \
-                     "${UBUNTU_IMAGE}.gz" \
+                     "gz" \
```

Comment:
I think we can get rid one argument here by updating the constants, e.g

```
UBUNTU_SERVER_RPI_IMAGE="ubuntu-18.04.4-preinstalled-server-armhf+raspi3.img.xz"
UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/bionic/release/${UBUNTU_SERVER_RPI_IMAGE}"
```

Reply:
Stretch: 

We could actually reduce it to one,

```
UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/bionic/release/ubuntu-18.04.4-preinstalled-server-armhf+raspi3.img.xz"

--disk-image $(basename ${UBUNTU_SERVER_RPI_IMAGE_URL})
```

Reply:
I like this